### ISSUE TITLE
Atom deprecated call warning:

### ISSUE THREAD

**Issue by jm3** [fkfk/vim-modeline#5]
on **Atom 1.13.0**, the built-in Deprecation Cop warns on file open:

**Use atom.textEditors.setGrammarOverride(editor, scopeName) instead**

```
GrammarRegistry.setGrammarOverrideForPath - /Atom.app/Contents/Resources/app.asar/src/grammar-registry.js:139:12
Object.setFileType - ~/.atom/packages/vim-modeline/lib/vim-modeline.coffee:251:23
Object.detectAndApplyModelineSetting - ~/.atom/packages/vim-modeline/lib/vim-modeline.coffee:140:14
<unknown> - ~/.atom/packages/vim-modeline/lib/vim-modeline.coffee:74:24
<unknown> - /Atom.app/Contents/Resources/app.asar/src/workspace.js:334:16
Function.module.exports.Emitter.simpleDispatch -
/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14
```

**Comment by fkfk** [comment]
I have fixed the problem in the v0.5.1.
Please update vim-modeline.

**Comment by jm3** [comment]
Thank you, closing!
